core/src/UI/AuthorizationResult.cs

```diff
@@ -87,6 +87,13 @@ internal AuthorizationResult(AuthorizationStatus status)
 
         public void ParseAuthorizeResponse(string webAuthenticationResult)
         {
+            if (!string.IsNullOrWhiteSpace(webAuthenticationResult) &&
+                webAuthenticationResult.StartsWith(Constants.FormPostPrefix, StringComparison.OrdinalIgnoreCase))
+            {
+                this.Code = webAuthenticationResult.Substring(Constants.FormPostPrefix.Length);
```

Comment:
>this.Code [](start = 16, length = 9)

I'm personally in favor of this. seems though the rest of this file doesn't really use it. consider following the existing pattern (or update all)

Reply:
I have removed this part of the code, ensuring that a url is returned instead.